europeana-eventlistener/src/main/java/eu/europeana/keycloak/user/UserRemovedConfig.java

```diff
@@ -11,33 +11,54 @@
-
-    public static final String SET_API_URL = "https://set-api-test.eanadev.org/set/";
-
-
-    public static final  String ERROR_ICON                    = ":x:";
-    public static final  String ERROR_ASCII                   = "✘";
-    public static final  String OK_ICON                       = ":heavy_check_mark:";
-    public static final  String OK_ASCII                      = "✓";
-    public static final  String SLACK_USER_DELETE_MESSAGEBODY =
-        "{\"text\":\"On %s, user %s has requested to remove their account.\\n" +
-        "This has just been done automatically for those systems marked with :heavy_check_mark: :\\n\\n" +
-        "[%s] Keycloak\\n" + "[%s] The User Sets API\\n" + "[:x:] The recommendation engine\\n" +
-        "[:x:] Mailchimp\\n\\n" +
-        "From the remaining systems (marked with :x: above) their account should be removed within 30 days (before %s).\"}";
-    private static final String REQUEST_RECEIVED              = "A problem prevented sending a Delete User Account notification for user ID %s to Slack.";
-    private static final String NO_ACTION_BUT_LOGGED          = "carrying out this request.\\nNo action was taken.\\nThe user token has been logged in Kibana.";
-    private static final String NO_ACTION_BUT_LOGGED_PERIOD   = NO_ACTION_BUT_LOGGED + "\"}";
-
-    public static final  String SLACK_SERVICE_UNAVAILABLE_MESSAGEBODY =
-        REQUEST_RECEIVED + "a server error occurred which prevented " + NO_ACTION_BUT_LOGGED_PERIOD;
-    private static final String NO_ACTION_LOGGED_AND_ERROR            = NO_ACTION_BUT_LOGGED + "\\n\\n[%s]\"}";
-    public static final  String SLACK_FORBIDDEN_MESSAGEBODY           =
-        REQUEST_RECEIVED + "an authorisation/authentication problem for the embedded Keycloak User prevented " +
-        NO_ACTION_LOGGED_AND_ERROR;
+    static final boolean DEBUG_LOGS     = System.getenv("DEBUG_LOGS").equalsIgnoreCase("true");
+    static final String GRANT_TYPE      = "password";
+    static final String CLIENT_ID       = System.getenv("CLIENT_ID");
+    static final String CLIENT_SECRET   = System.getenv("CLIENT_SECRET");
+    static final String SCOPE           = System.getenv("SCOPE");
+    static final String DELETE_MGR_ID   = System.getenv("DELETE_MGR_ID");
+    static final String DELETE_MGR_PW   = System.getenv("DELETE_MGR_PW");
+    static final String SET_API_URL     = System.getenv("SET_API_URL");
+    static final String AUTH_SERVER_URL = System.getenv("AUTH_SERVER_URL");
+    static final String OIDTokenURL     = AUTH_SERVER_URL + "realms/europeana/protocol/openid-connect/token";
+
+    static final String ERROR_ASCII                   = "✘";
+    static final String OK_ICON                       = ":heavy_check_mark:";
+    static final String OK_ASCII                      = "✓";
+    static final String SLACK_USER_DELETE_MESSAGEBODY = "{\"text\":\"On %s, user %s has requested to remove their " +
```

Comment:
wouldn't be better to keep only the configurations in this class and to move all constants for messages to a different class? (so we have a clear overview of configurations received from system properties)